job-server/src/main/scala/spark/jobserver/JobManagerActor.scala

```diff
@@ -53,7 +53,7 @@ object JobManagerActor {
 
   // Akka 2.2.x style actor props for actor creation
   def props(daoActor: ActorRef, supervisorActorAddress: String = "",
-      initializationTimeout: FiniteDuration = 30.seconds): Props =
+      initializationTimeout: FiniteDuration = 40.seconds): Props =
```

Comment:
Do you think we can have this in application.conf?

Reply:
I pushed another commit where I make initialization timeout configurable.